framework-plugins/lisk-framework-report-misbehavior-plugin/src/db.ts

```diff
@@ -12,10 +12,33 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { KVStore } from '@liskhq/lisk-db';
+import { formatInt, KVStore } from '@liskhq/lisk-db';
+import { codec } from '@liskhq/lisk-codec';
+import { RawBlockHeader } from '@liskhq/lisk-chain';
 import * as os from 'os';
 import { join } from 'path';
 import { ensureDir } from 'fs-extra';
+import { RegisteredSchema } from 'lisk-framework';
+import { hash } from '@liskhq/lisk-cryptography';
+
+export const blockHeadersSchema = {
+	$id: 'lisk/reportMisbehavior/blockHeaders',
+	type: 'object',
+	required: ['blockHeaders'],
+	properties: {
+		blockHeaders: {
+			type: 'array',
+			fieldNumber: 1,
+			items: {
+				dataType: 'bytes',
+			},
+		},
+	},
+};
+
+interface BlockHeaders {
```

Comment:
can this be a type?

Reply:
i think lint will complain because it's a object format